### PR TITLE
doc: working_with: changelog for recent edits

### PR DESCRIPTION
**Pull request by greg-fer**
Added changelog entries for recent edits to Working with guides. Re PRs #15598, #15139, #15661, #15708, #15596.
Added changelog entry for edits about sysbuild. Re PR #15830.

----

- [x] Rebase after PR #15830 is in

### CODE REVIEW

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst [hunk 1]
```diff
@@ -68,24 +71,27 @@ Working with nRF54L Series
 Working with nRF53 Series
 =========================
 
+* Restructured the contents of this section into multiple pages, with :ref:`ug_nrf53` as the landing page.
+* Added the :ref:`features_nrf53` page.
 * Moved :ref:`ug_nrf5340_gs` to the :ref:`gsg_guides` section.
+* Changed the default value for the Kconfig option :kconfig:option:`CONFIG_CLOCK_CONTROL_NRF_ACCURACY` from 500 to 250 if :kconfig:option:`CONFIG_CLOCK_CONTROL_NRF_K32SRC_RC` is used.
```
Comment (divipillai) [inline doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:77]:
```suggestion
* Changed the default value for the Kconfig option :kconfig:option:`CONFIG_CLOCK_CONTROL_NRF_ACCURACY` from ``500`` to ``250`` if :kconfig:option:`CONFIG_CLOCK_CONTROL_NRF_K32SRC_RC` is used.
```

These sentences (under Working with nRF53 Series) could be in the Added and updated bullet points.

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst [hunk 2]
```diff
@@ -52,40 +53,48 @@ Build and configuration system
...
 ==========================
 
 * Added the :ref:`ug_nrf54l15_gs` page.
-* Changed the default value for the Kconfig option :kconfig:option:`CONFIG_CLOCK_CONTROL_NRF_ACCURACY` from 500 to 250 if :kconfig:option:`CONFIG_CLOCK_CONTROL_NRF_K32SRC_RC` is used.
+* Changed the default value for the Kconfig option :kconfig:option:`CONFIG_CLOCK_CONTROL_NRF_ACCURACY` from ``500`` to ``250`` if :kconfig:option:`CONFIG_CLOCK_CONTROL_NRF_K32SRC_RC` is used.
 
 Working with nRF53 Series
 =========================
 
+* Restructured the contents of this section into multiple pages, with :ref:`ug_nrf53` as the landing page.
```
Comment (umapraseeda) [inline doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:74]: Should we move this under Updated (wherever applicable)?

**Review by umapraseeda — APPROVED** [review]

**Review by divipillai — APPROVED** [review]

**Comment by shanthanordic** [comment]
@greg-fer  can this be closed as the changelog will go with barteks PR ?

**Comment by b-gent** [comment]
@greg-fer  please verify in https://github.com/nrfconnect/sdk-nrf/pull/15961 that all has been moved correctly.

**Pull request closed without merging**